### PR TITLE
Met236/prevent weekend rides

### PR DESCRIPTION
**Pull request by michaeltorku**
Used an alert to prevent users from scheduling rides in the carriage frontend on the weekends.

### CODE REVIEW

**Review by tuckerstanley — DISMISSED** [review]
The code looks very good for the most part. It definitely does the functionality that you were hoping for in not letting the user choose rides on the weekend. My general feedback consists of removing the unnecessary import statements (you dont need import moment in most files now because you deleted that code), and maybe considering using a toast instead of an alert to display the error. This might look better, however check with Matt about that. Making an extra method for Format_date was very smart, and your code looks good.

File: frontend/src/components/UserDetail/UserDetail.tsx [hunk 1]
```diff
@@ -133,14 +133,14 @@ const UserDetail = ({
...
-            {!isRider && (
+            {/* {!isRider && (
               <input
                 type="image"
                 className={styles.editIcon}
                 alt="trash"
                 src={detailTrash}
               />
-            )}
+            )} */}
```
[the remark this answers is not in the document]
Comment (michaeltorku) [reply frontend/src/components/UserDetail/UserDetail.tsx:143]: That was there but not by me, I'll delete it

**Comment by michaeltorku** [comment]
All the changes have been made as requested. Feel free to review again

**Review by mg2239 — CHANGES_REQUESTED** [review]
really nice work, thanks for making all the changes! added a couple comments, and in case you missed my previous message, the Rider Web ride modal also needs these restrictions too. the code should be pretty much a copy and paste though 👍

**Review by mg2239 — APPROVED** [review]
looks good, great work!

**Pull request merged**